hourglass.rb

```diff
@@ -12,5 +12,4 @@ def hourglassSum(hour)
 end
 
 arr = [1,2,3,4,5,6]
-puts hourglassSum([arr, arr, arr, arr, arr,arr])
-
+puts hourglassSum([arr, arr, arr, arr, arr, arr])
```

Comment:
Why are we passing same arr so many times?

Reply:
Just giving as the input array of arrays